vertexai/internal/sentencepiece/token.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2024 Google LLC
```

Comment:
This probably isn't worth its own file, but up to you.

Reply:
Originally I thought there'd be more methods but also felt wrong to define it in `encode.go` because I also want a decoder and `Token` would be shared